### ISSUE TITLE
Support for symbols (ruby), atoms (elixir) and keys in general

### ISSUE THREAD

**Issue by joaoevangelista** [dracula/visual-studio-code#38]
Since symbols, atoms and keys (`:foo`, `bar:  "a value"`, `key: :atom`) are important part of the languages (ruby, elixir, maybe others?) they could have a dedicated highlight color instead of plain white, which does not provide any relevance for the syntax highlight, e.g.:

Using Code 1.12.2 & Dracula 1.17.0

Elixir:
<img width="340" alt="ex" src="https://cloud.githubusercontent.com/assets/5256711/26214170/c03f66b2-3bd1-11e7-8688-dff2bfa4674a.PNG">

Ruby:
<img width="229" alt="ruby" src="https://cloud.githubusercontent.com/assets/5256711/26214314/4e27d52c-3bd2-11e7-9a61-fbf4a443b53f.PNG">

Possible Solutions
----

Atom theme uses the purple color to keys in general such as:

<img width="321" alt="ex-atom" src="https://cloud.githubusercontent.com/assets/5256711/26214409/b801b864-3bd2-11e7-9461-3f9960583bc5.PNG">

Which would be enough to have.
But since the issue #12 I think would make more sense use the new "keys" highlight as the yaml keys making them more distinguishable.

<img width="229" alt="ruby" src="https://camo.githubusercontent.com/b577bb5daabd571eb817f91d3d1af23fb15de159/68747470733a2f2f692e696d6775722e636f6d2f364a764c736b492e706e67">

**Comment by dsifford** [comment]
@joaoevangelista Yep, my thoughts mirror yours. Getting everything uniform is on my todo list.

The plan for now is to make sure all the core languages match the most complete language that I have done (typescript) and then once that's done, I'll probably go in and spot-fix things as needed.

I'm going to probably start a scope->color mapping list in #12 when I get some free time and can commit a good 2-3 hours on it.

Please chime in there with your thoughts/suggestions!

Gonna leave this issue open for now as well because your screenshots are helpful 😄

**Comment by dsifford** [comment]
@joaoevangelista Here's what I think the solution is going to come down to....

![image](https://cloud.githubusercontent.com/assets/5240018/26516852/4438a0c0-425b-11e7-9513-008f361afa5e.png)

(In case you missed it, I added the pink color to the colon)

In this particular example, I agree that maybe the cyan colored key would look better. But globally, it's a bit too much for non-data languages (e.g. not yaml, json, toml, etc)... The pattern that works best across all files is to make keys the white color and also keep the variables the white color.

Also, while doing that, I noticed quite a handful of issues with the Ruby tmLanguage file (one example being `puts` isn't scoped to anything, which is strange).

Gonna roll this out when I roll out the **big** update later this weekend.
